CMakeLists.txt

```diff
@@ -108,10 +108,11 @@ link_directories(${CCD_LIBRARY_DIRS})
 option(FCL_WITH_OCTOMAP "octomap library support" ON)
 set(FCL_HAVE_OCTOMAP 0)
 if(FCL_WITH_OCTOMAP)
-  if(PKG_CONFIG_FOUND)
+  find_package(octomap QUIET)
+  if(NOT OCTOMAP_INCLUDE_DIRS AND NOT OCTOMAP_LIBRARY_DIRS AND PKG_CONFIG_FOUND)
```

Comment:
True, though this would allow you to specify `OCTOMAP_INCLUDE_DIRS` and `OCTOMAP_LIBRARY_DIRS` as cache variables if you are using an old version without the necessary config.